### ISSUE TITLE
Plain text field accessibility

### ISSUE THREAD

**Issue by missmatsuko** [craftcms/cms#6914]
### Description
The user should be able to tell if there is a maximum character count, and whether the field can accept line breaks.

- Add back the corner notch for textareas (I think there was a regression from #4030)
- Field instructions should be programmatically linked to the field with `aria-describedby`
- Use aria-live regions to announce character count when the field is first entered, and when the user stops typing in the field. Some examples:
  - [GOV.UK Design System: Character count](https://design-system.service.gov.uk/components/character-count/)
  - [aria-atomic & aria-relevant on aria-live regions](http://pauljadam.com/demos/aria-atomic-relevant.html)

### Additional info

- Craft version: 3.5.11.1
- PHP version:
- Database driver & version:
- Plugins & versions:

**Comment by brandonkelly** [comment]
This was actually already reported by #6872, and fixed for the next release :)

**Comment by brandonkelly** [comment]
#7821 is merged in for the next release 🎉
